src/runtime/components/MarkdownRenderer.ts

```diff
@@ -106,58 +106,86 @@ export default defineComponent({
 /**
  * Render a markdown node
  */
-function renderNode (node: MarkdownNode, h: CreateElement, documentMeta: ParsedContentMeta): ContentVNode {
-  const originalTag = node.tag
-  // `_ignoreMap` is an special prop to disables tag-mapper
-  const renderTag: string = (typeof node.props?.__ignoreMap === 'undefined' && documentMeta.tags[node.tag]) || node.tag
-
+function renderNode (node: MarkdownNode, h: CreateElement, documentMeta: ParsedContentMeta, parentScope: any = {}): ContentVNode {
   /**
    * Render Text node
    */
   if (node.type === 'text') {
     return h(Text, node.value)
   }
 
+  const originalTag = node.tag!
+  // `_ignoreMap` is an special prop to disables tag-mapper
+  const renderTag: string = (typeof node.props?.__ignoreMap === 'undefined' && documentMeta.tags[originalTag]) || originalTag
+
+  if (node.tag === 'binding') {
+    return renderBinding(node, h, documentMeta, parentScope)
+  }
+
   const component = resolveVueComponent(renderTag)
   if (typeof component === 'object') {
     component.tag = originalTag
   }
 
+  const props = propsToData(node, documentMeta)
   return h(
     component as any,
-    propsToData(node, documentMeta),
-    renderSlots(node, h, documentMeta)
+    props,
+    renderSlots(node, h, documentMeta, { ...parentScope, ...props })
   )
 }
 
+function renderBinding (node: MarkdownNode, h: CreateElement, documentMeta: ParsedContentMeta, parentScope: any = {}): ContentVNode {
+  const data = {
+    ...parentScope,
+    $route: () => useRoute(),
+    $document: documentMeta,
+    $doc: documentMeta
+  }
```

Comment:
We should define what should be available in interpolation.

Reply:
I think this is a good start. Also we will need to see how we can support this in the editor 🤔